.env.docker.dev

```diff
@@ -8,13 +8,13 @@ DEBOUNCE_SECONDS=10
 # Configure the amount of capacity Graph Service can use.
 CAPACITY_LIMIT={"type":"percentage", "value":80}
 
-# Add the graph environment type. This can be 'Dev' or 'Rococo' or 'Mainnet'.
-GRAPH_ENVIRONMENT_TYPE=Dev
+# Add the graph environment type. This can be 'Dev' or 'TestnetPaseo' or 'Mainnet'.
+GRAPH_ENVIRONMENT_TYPE=Mainnet
 
 # [NOTE]: The following config is only used for Dev environments.
 # Add the graph environment config in JSON format only used for Dev environments.
 # Be careful to escape any inner quotes as this is in a .env file.
-GRAPH_ENVIRONMENT_DEV_CONFIG={"sdkMaxStaleFriendshipDays": 100,"maxPageId": 100,"dsnpVersions": ["1.0"],"maxGraphPageSizeBytes": 100,"maxKeyPageSizeBytes": 100,"schemaMap": {"1": {"dsnpVersion": "1.0","connectionType": "follow","privacyType": "public"},"2": {"dsnpVersion": "1.0","connectionType": "follow","privacyType": "private"},"3": {"dsnpVersion": "1.0","connectionType": "friendship","privacyType": "private"}},"graphPublicKeySchemaId": 4}
+# GRAPH_ENVIRONMENT_DEV_CONFIG='{"sdkMaxUsersGraphSize":10000,"sdkMaxStaleFriendshipDays":90,"maxGraphPageSizeBytes":1024,"maxPageId":32,"maxKeyPageSizeBytes":65536,"graphPublicKeySchemaId":7,"schemaMap":[[8,["1.0",{"connectionType":"follow","privacyType":"public"}]],[9,["1.0",{"connectionType":"follow","privacyType":"private"}]],[10,["1.0",{"connectionType":"friendship","privacyType":"private"}]]],"dsnpVersions":["1.0"]}'
```

Comment:
Can probably just delete this; nobody should be playing with Dev configs unless they're really doing GraphSDK development.

Eventually we need to update GraphSDK so that the schema IDs are not part of the config for dev, but just get passed in to the constructor, optionally, as detected from the chain, if not using a standard image.

Reply:
I wasn't sure, but I think it is reasonable to just remove.

Will update